utils/results.py

```diff
@@ -42,6 +58,27 @@ def load_final_sumstats_mt(filter_phenos: bool = True, filter_variants: bool = T
         ).drop('pheno_indices')
         mt = mt.filter_cols(hl.len(mt.pheno_data) > 0)
 
+    if filter_pheno_h2_qc:
+        mt = mt.filter_cols(mt.pheno_data.any(lambda x: (~hl.is_missing(x.heritability.qcflags.pass_all)) & \
```

Comment:
not `hl.is_defined`?

Reply:
Changed ~hl.is_missing to hl.is_defined (these are identical I think right?)